docs/src/index.md

```diff
@@ -193,7 +193,292 @@ Since we haven't created our own project yet, we are in the default project, loc
 To return to the `julia>` prompt, either press backspace when the input line is empty or press Ctrl+C.
 Help is available by calling `pkg> help`.
 
-To generate files for a new project, use `pkg> generate`.
+
+### Adding packages
+
+There are two ways of adding packages, either using the `add` command or the `dev` command.
+The most frequently used one is `add` and its usage is described first.
+
+#### Adding registered packages
+
+```
+(v0.7) pkg> add Example
+   Cloning default registries into /Users/kristoffer/.julia/registries
+   Cloning registry Uncurated from "https://github.com/JuliaRegistries/Uncurated.git"
+  Updating registry at `~/.julia/registries/Uncurated`
+  Updating git-repo `https://github.com/JuliaRegistries/Uncurated.git`
+ Resolving package versions...
+  Updating `~/.julia/environments/v0.7/Project.toml`
+  [7876af07] + Example v0.5.1
+  Updating `~/.julia/environments/v0.7/Manifest.toml`
+  [7876af07] + Example v0.5.1
+  [8dfed614] + Test
+```
+
+Since this is the first time adding anything, the [default registry](https://github.com/JuliaRegistries/Uncurated) is cloned.
+The status update shows a short form of the package UUID to the left, then the package name, and the version.
+Since standard libraries (e.g. `Test`) are shipped with Julia, they do not have a version. The project status contains the packages
+you have added yourself, in this case, `Example`. The manifest status, in addition, includes the dependencies of explicitly
+added packages.
```

Comment:
Maybe show the output of `pkg> status` directly after `The project status contains...`.

Same for `The manifest status...`, but show the actual manifest status (is that just the content of the `Manifest.toml`)?